tests/test_common.sh.in

```diff
@@ -231,7 +231,7 @@ set_chroot_offline_test_mode() {
 		return
 	fi
 	if test -x "$OSCAP_CHROOTABLE_EXEC"; then
-		if ! getcap "$OSCAP_CHROOTABLE_EXEC" | grep -q 'cap_sys_chroot+ep'; then
+		if ! getcap "$OSCAP_CHROOTABLE_EXEC" | grep -q -P 'cap_sys_chroot[=+]+ep'; then
```

Comment:
Why is the second `+` there? It's either `+` or `=` but not a permutation.

```suggestion
		if ! getcap "$OSCAP_CHROOTABLE_EXEC" | grep -q -P 'cap_sys_chroot[=+]ep'; then
```

Reply:
From `cap_to_text(3)`:

> A  textual  representation of capability sets consists of one or more whitespace-separated clauses.  Each clause specifies some operations on a capability set; the set starts out with all capabilities lowered, and the meaning of the string is the state of the capability set after all the clauses have been applied in order.

> Each clause consists of a list of comma-separated capability names (or the word 'all'), followed by an action-list.  An action-list consists of a sequence of operator flag pairs.  Legal operators are: '=', '+', and '-'. Legal flags are: 'e', 'i', and 'p'. These flags are case-sensitive and specify the Effective, Inheritable and Permitted sets respectively.

> The  action  list  can  consist  of  multiple  operator  flag  pairs;  the  actions are performed in left-to-right order.  Thus, for example, "cap_fowner+p-i" is equivalent to "cap_fowner+p cap_fowner-i".  As another example, "cap_fowner+pe-i" and "cap_fowner=+pe" are equivalent.

Reply:
So, while `==` and `++` combinations might not be valid (I'm not sure here, BTW), the `=+` combination is.

Reply:
I'm even thinking about `cap_sys_chroot[=+](ep|pe)`

Reply:
Thank you very much for finding this!